### PR TITLE
Create a PowerShell Docker image.

### PR DESCRIPTION
**Pull request by lalyos**
This Dockerfile makes very easy to get started with PowerShell on linux.

You can try my public image:

```
docker run -it lalyos/powershell
```

Once its merged, it can be build under the Microsoft DockerHub user, so it will be

```
docker run -it microsoft/powershell
```

### CODE REVIEW

**Comment by daxian-dbw** [comment]
This is cool! Thanks @lalyos!
@andschwa can you please review this?

**Comment by andschwa** [comment]
Hey this is great!

Just FYI (I know I don't have an issue for it yet, it's been a crazy week), I intend to write Dockerfiles for each distro that are setup to build (and package) PowerShell too. I'll probably move this once I do so; but definitely merging!

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:16.04
+
+RUN apt-get update && apt-get install -y curl  libunwind8 libicu55
+RUN curl -LO https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-alpha.9/powershell_6.0.0-alpha.9-1ubuntu1.16.04.1_amd64.deb
```
Comment (andschwa) [inline Dockerfile]: This is hard-coded; but totally a-okay for now. We can do more clever stuff later.

File: docs/installation/docker.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+Docker
+======
```
Comment (andschwa) [inline docs/installation/docker.md]: I can tell you follow existing conventions; *thank you*.

**Pull request merged**